### DISCUSSION TITLE
[PCQM4Mv2] Question on modifying data-loader & deriving new features

### DISCUSSION THREAD

**Discussion by sajaddarabi** [snap-stanford/ogb#384] [PCQM4M-LSC]
Hi OGB-LSC Team,

On the rules page it mentions: "We do not allow the use of any external datasets to train models"

I would like to have some clarification on this, would this include deriving additional features based on the smiles strings considered adding external data? Perhaps to make it more concrete which of the following would be considered okay:

- Adding a feature that is not present in the original data-loader but derived from the molecule/compound for example number of charged atoms, or suppose chirality wasn't in the atom feature set, and I added this.
- Generating conformers from the 2d-smiles string? 
- How about computing Humo-Lumo GAP for these conformers?

Thanks

**Comment by weihua916** [comment]
Hi! Great questions!

You can really add any features to the training molecules. On test molecules, you can also add any features but you need to obey the computational budget rule (see [here](https://ogb.stanford.edu/docs/lsc/rules/)).  

Let me know if you have any questions!

**Reply by sajaddarabi** [reply]
Got it, thanks!
